src/sentry/incidents/subscription_processor.py

```diff
@@ -444,9 +445,8 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
             # TODO: Delete subscription here.
             return
 
-        # TODO:
-        # if AlertRule.monitor_type === ACTIVATED and AlertRule.is_expired():
-        #       delete query in snuba
+        # Trigger callbacks for any AlertRules that may need to know about the subscription update
+        invoke_alert_subscription_callback(self.alert_rule.monitor_type, self.subscription)
```

Comment:
ohh i see. the `monitor_type` on the alert_rule here will be an `int`

so lets just keep the registry key'd to an int as well

Reply:
chatted in slack, resolution is to remove the `| int` and just use the enum instance for now.